tests/gordo_components/builder/test_builder.py

```diff
@@ -11,15 +11,21 @@
 import pytest
 import numpy as np
 
+from sklearn.multioutput import MultiOutputRegressor
+from sklearn.linear_model import LinearRegression
+
 import gordo_components
 from gordo_components.builder.build_model import (
     _save_model_for_workflow,
     provide_saved_model,
     _get_metadata,
+    _determine_offset,
 )
 from gordo_components.builder import build_model
 from gordo_components.dataset.sensor_tag import SensorTag
 
```

Comment:
Is there a extra line here? And if so, is it necessary?

Reply:
Line 26? Nope, it would be green if it was added. :thinking:

Reply:
Yeah, I would assume, but the ```from gordo_components.model import models``` is added at line 27, no?